config/config.php

```diff
@@ -539,103 +539,103 @@
+Amz51YNM0rGOJQKykfJRoM1ao2s7E12nCX3pH3b8G23aPCe69Gw5fffmnc1v5+m4
+1wg5L+qVIrB/TDjIBrDVs+wtPC8R3Pxhw74nPvt3CVGsEA+rRFz1PHMOqAdMj/zI
+M+291UdIaiMHzZXvPht1Z/ptMeeyMnAxnbuiafh8eR389p1m5K9RfpRbJmglXx6M
+J7ly448K8111qc92KDyGxTMvCBnQA8XKLd8TkYSGU2vbQYnGvmZKK6urUDVP6gJ4
+whlUk4Etx1/kqDqeKF/2Uw==',
 			],
-	],
+		],
 
 		'21.0.8' => [
 			'100' => [
-				'latest' => '22.2.4 RC2',
-				'internalVersion' => '22.2.4.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-22.2.4rc2.zip',
+				'latest' => '22.2.4 RC3',
+				'internalVersion' => '22.2.4.2',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-22.2.4rc3.zip',
 				'web' => 'https://docs.nextcloud.com/server/22/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
 				'minPHPVersion' => '7.3',
-				'signature' => 'FXCaKtTs1RdSndCPVkbw4PG9562KFNCveYy+IVm4LaGF9U30CwlHGn2/9gIW8phK
-TF8Kg/jzsQi2WrHw0R+4vQb+S7AsFhu7wOInGDeT5UC14iLtKFOzoZyaFuO3LcmM
-9Om8wuTJhwWGlum0sn2zuQWVfsOjCLiJjip4Ogl4O473yNfscicAS4HVFvTIoxmh
-ypgeb2JCoOhbFTm04MS4ftqePlc9RqlfyeNmJ+gyq80mAwIEANBnvUBfOfM95a7c
-q/2qlUKWj3Gb0GZLtnUnGuByOAZhgV7gONiBj6rcKnAuBEzlEMNcKO5efHXTr6tl
-IZCTzrIZfwjmWGgPjZDLiQ==',
+				'signature' => 'vxmfiPafdBj9t0HPqOlfmptv1VcKGFJL3S7Oksj88c84fCMSXdvPqM+mkjsIuad5
+Amz51YNM0rGOJQKykfJRoM1ao2s7E12nCX3pH3b8G23aPCe69Gw5fffmnc1v5+m4
+1wg5L+qVIrB/TDjIBrDVs+wtPC8R3Pxhw74nPvt3CVGsEA+rRFz1PHMOqAdMj/zI
+M+291UdIaiMHzZXvPht1Z/ptMeeyMnAxnbuiafh8eR389p1m5K9RfpRbJmglXx6M
+J7ly448K8111qc92KDyGxTMvCBnQA8XKLd8TkYSGU2vbQYnGvmZKK6urUDVP6gJ4
+whlUk4Etx1/kqDqeKF/2Uw==',
 			]
 		],
 
 		'21' => [
 			'100' => [
-				'latest' => '21.0.8 RC2',
+				'latest' => '21.0.8 RC3',
 				'internalVersion' => '21.0.8.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-21.0.8rc2.zip',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-21.0.8rc3.zip',
 				'web' => 'https://docs.nextcloud.com/server/21/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
 				'minPHPVersion' => '7.3',
-				'signature' => 'e4bjy2g4RL5BjQCbHK9uV68nelgNgQZzyDGZ3SBSaVjpd9zKRuFde/OBiTskF8ke
-pzsZy/cSy83FVXOE3ZQa5iofWQ/rp4BcpJFbBixfnHYoZpmrQsckwI0OxXNGHEf0
-04SrF1SaNt4fhz0pycAUEdpIDu9I1jXVuimFvbom6c73hKfBNKJbwy2P8OQd+L3M
-PINkz0oEjFPcbjt2vQ1MaJ4DdmfijBWBa0Q5EdzZLtnVQCNtqIUahLQJcLh/IVYT
-WWyW4B2J0+5KcWfS+9m4g3FQxt3qoz63K96sE0M+/giwTWQQHe6Mw+X+IOkJFtxm
-tT1qSks6Ix8dsbNGcugwCA==',
+				'signature' => 'eG4iWgMHpp5PD2jGdDeTMuLKTrpgvqikt4T40xeafepe/gK+V3Y7A8MbvGMBMnSh
+KEYF03P2Qwybo1odTfCwAdSDezYBewq5VBk0HhQf2YybWEnHuGZDh39dt+xcH18z
+06Tc05qnUVCYnVmm+KgVcLJr+J82kxZ+PnFnEHCIo9vRXzfWWMd9kKw3VQjKnDSR
+KRcjEYOsG0GNsR5MD111zvr/KYE/mlJjUJ5UtprokS90OnGpEAdGcmKA27PaKhWO
+9psB9VX3BARdSiCT6qZbPmBZHpY9q+7YurYjJef0DBAmHWZ40RKRT0b44uEEytk7
+96rmSG0T48tzZaonZBXmgA==',
 			],
 		],
 
 		'20.0.14' => [
 			'100' => [
-				'latest' => '21.0.8 RC2',
+				'latest' => '21.0.8 RC3',
 				'internalVersion' => '21.0.8.1',
```

Comment:
same here, shouldn't the version be different?

Reply:
No, we didn't change in between.

Reply:
We blocked rc2, so I figured it was not need to

Reply:
That means people having 21.0.8 rc2 will not be offered the update to rc3
The numeric version must always increase

Reply:
We did not offered 21.0.8rc2

Reply:
https://github.com/nextcloud/updater_server/pull/687

Reply:
okay, right fine then.
Would just make it a fixed rule to be safe rather than sorry